src/app/core/header/header.component.html

```diff
@@ -1,14 +1,17 @@
 <mat-toolbar class="ccf-header">
-  <mat-icon class="logo" aria-hidden="false" aria-label="Reset tool" matTooltip="Human BioMolecular Atlas Project"
-            matTooltipPosition="right" svgIcon="app:logo" (click)="logoClicked.emit()"></mat-icon>
-
+  <a class="portal-home" href="dataSourceService.dbOptions.hubmapDataUrl">
```

Comment:
This href seems wrong? Shouldn't it be a reference to the hubmap portal page? @bherr2 Can you shed some light on this?

Reply:
Yes, it should be. Depending on the environment, the portal url will be different, thus it needs to be injected. It shouldn't be hubmapDataUrl, but hubmapPortalUrl -- ie https://github.com/hubmapconsortium/ccf-ui/blob/develop-v2/projects/ccf-database/src/lib/ccf-database.ts#L25